src/frontend/src/components/App.js

```diff
@@ -2,14 +2,16 @@ import React from 'react';
 import Menuitems from './Menuitems';
 import Header from './Header';
 import Footer from './Footer';
+import Mainpage from './Mainpage';
 import 'bootstrap/dist/css/bootstrap.min.css';
 
 class App extends React.Component {
   render() {
     return (
       <div>
         <Header />
-        <Menuitems />
+        {/*<Menuitems />*/}
+        <Mainpage></Mainpage>
```

Comment:
Can you make this self closing like the ones above it?

Reply:
Fixed, thank you.